sregistry/database/models.py

```diff
@@ -144,6 +145,12 @@ def init_db(self, db_path):
     self.database = 'sqlite:///%s' % db_path
     self.storage = SREGISTRY_STORAGE
 
+    if not os.path.exists(SREGISTRY_BASE):
+        bot.exit("Database location {} does not exist.".format(SREGISTRY_BASE))
```

Comment:
This will work in most cases, but it will poop out if the user calls the function manually with a custom / different path. I think we would want to first do a check for None, and then derive the parent folder directly from `db_path`. What do you think?

Reply:
I thought about "being pythonic" and doing a try, except (ask for forgiveness not permission) but then the user wouldn't be told specifically if it didn't exist, or was a permissions error (and they would have to debug). This way has more lines of code, but it tells the user specifically what's up.

Reply:
Hi @vsoch,

I am on leave for the rest of the week so won't be able to look into this properly. If you can't wait until Monday, feel free to fix it yourself 😄

Reply:
I can wait, but I'd be happy to fix it up for you so we both can rest easily. Enjoy your leave!